src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -182,21 +190,24 @@ describe('NameAndPictureScreen', () => {
     expect(getByText('nameAndPicGuideCopyTitle')).toBeTruthy()
     expect(getByText('nameAndPicGuideCopyContent')).toBeTruthy()
   })
+
   it('does not render skip button when configured so', () => {
     // TODO replace route param with mock Statsig flag
+    // StatSigFlags.shallSkipUsername = jest.fn().mockReturnValue(true)
```

Comment:
in general, we should not include commented out code in our PR's against the `main` branch